lib/signs/bus.ex

```diff
@@ -456,6 +478,20 @@ defmodule Signs.Bus do
     ]
   end
 
+  defp add_preamble([], _), do: []
+
+  # Special case: South Station is currently the only terminal we handle, so it should say
+  # "departures" instead of "arrivals".
```

Comment:
Hmm interesting.. is this actually true? I think we also have PA/ESS at the Forest Hills busway and Nubian, at least. Or are those not necessarily terminals for some reason? 🤔

Reply:
I rechecked, TWE only says "departures" at South Station (and also Harvard, but that's a special case I haven't looked into fully yet). But I don't know if that's the behavior we want going forward, so I'll check.